src/StmtToHTML.cpp

```diff
@@ -2099,7 +2162,33 @@ class HTMLCodePrinter : public IRVisitor {
         print_text("(");
         print_type(op->type);
         print_text(")");
-        print_function_call("vector_reduce", {op->op, op->value});
+        std::string op_str;
+        switch (op->op) {
```

Comment:
This should use the operator<< already in IRPrinter.h (and the implementation in IRPrinter.cpp can be changed to be snake case if you like).